charts/incubator/speedtest-tracker/questions.yaml

```diff
@@ -43,49 +42,45 @@ questions:
 # Include{serviceList}
 # Include{persistenceRoot}
         - variable: config
-          label: App Config Storage
-          description: Stores the Application Configuration.
+          label: "App Config Storage"
+          description: "Stores the Application Configuration."
           schema:
-            additional_attrs: true
```

Comment:
Why is that removed?

Reply:
Reverted this not sure why I took I took it out...